python/lsst/sims/maf/utils/opsimUtils.py

```diff
@@ -95,13 +96,16 @@ def getFieldData(opsimDb, sqlconstraint):
         fieldData = opsimDb.fetchFieldsFromSummaryTable(sqlconstraint)
     return fieldData
 
-def getSimData(opsimDb, sqlconstraint, dbcols, stackers=None):
+def getSimData(opsimDb, sqlconstraint, dbcols, stackers=None, tableName='Summary', distinctExpMJD=True,
+               groupBy='expMJD'):
     """
     Query the database for the necessary simdata columns, run any needed stackers,
     return simdata array.
+
     """
     # Get data from database.
-    simData = opsimDb.fetchMetricData(dbcols, sqlconstraint)
+    simData = opsimDb.fetchMetricData(dbcols, sqlconstraint, tableName=tableName,
+                                      distinctExpMJD=distinctExpMJD, groupBy=groupBy)
```

Comment:
We should think about this 'distinctExpMJD' + groupBy expMJD thing. Guess it can be a separate ticket though.